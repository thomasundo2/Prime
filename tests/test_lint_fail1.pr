int main()
{
    /* for pow we raise by unsigned int */
    printl(21973469182365874353456l ^ 2l);
    return 0;
}