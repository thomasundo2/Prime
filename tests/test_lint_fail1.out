Fatal error: exception Failure("illegal binary operator lint ^ lint in 21973469182365874353456 ^ 2")